extensions/amp/content/src/content/jcr_root/apps/core/wcm/extensions/amp/components/search/v1/search/.content.xml

```diff
@@ -19,5 +19,5 @@
     jcr:description="Full-text search with AMP support"
     jcr:primaryType="cq:Component"
     jcr:title="Quick Search (AMP v1)"
-    sling:resourceSuperType="core/wcm/components/search/v1/search"
+    sling:resourceSuperType="core/wcm/components/search/v2/search"
```

Comment:
shouldn't we create a new AMP search v2 version instead? This might break existing applications using the AMP search v1 component.